dotnet/src/SemanticKernel.Abstractions/Contents/AnnotationContent.cs

```diff
@@ -12,25 +13,60 @@ namespace Microsoft.SemanticKernel.Agents;
 public class AnnotationContent : KernelContent
 {
     /// <summary>
-    /// The file identifier.
+    /// The referenced file identifier.
+    /// </summary>
+    [JsonIgnore]
+    [Obsolete("Use `ReferenceId` property instead.  This method will be removed after June 1st 2025.")]
```

Comment:
No ETAs on messages.

```suggestion
    [Obsolete("Use `ReferenceId` property instead.")]
```

Reply:
Removed.

> NOTE: Wesley and Mark used this ETA for Obsolete messaging, so I had copied that.